### PR TITLE
Backend/km users test

### PR DESCRIPTION
**Pull request by jKwibe**
## Description
-  Created test for the users
-  Add teardown to delete everything from the tables after running the tests.

### CODE REVIEW

File: code/api/src/modules/user/tests/query.test.js [hunk 1]
```diff
@@ -7,13 +7,32 @@ import express from 'express';
...
+import models from "../../../setup/models";
 
 describe("user queries", () => {
   // this variable needs to be outside of the beforeAll block
   // in javascript. In Ruby it wouldn't matter.
   let server;
+  let USER;
   // test setup
-  beforeAll(() => {
+  beforeAll(async () => {
```
Comment (RyN21) [inline code/api/src/modules/user/tests/query.test.js:20]: looks good!

File: code/api/src/modules/user/tests/query.test.js [hunk 2]
```diff
@@ -26,24 +45,39 @@ describe("user queries", () => {
...
+      .post('/')
+      .send({ query: `{ user(id: ${ USER.id }) { name email }}`})
+      .expect(200)
+
+    expect(response.body.data.user.name).toEqual(USER.name)
+    expect(response.body.data.user.email).toEqual(USER.email)
   })
 
-  xit("user - returns user with specific id", async() => {
+  it("user - an error when wrong id is passed", async() => {
```
Comment (RyN21) [inline code/api/src/modules/user/tests/query.test.js:73]: nice edge case

File: code/api/src/migrations/6-createOrdersTable.js [hunk 3]
```diff
@@ -1,5 +1,7 @@
 'use strict';
```
Comment (RyN21) [inline code/api/src/migrations/6-createOrdersTable.js:1]: These tests look real good. Nice job on them

**Review by RyN21 — APPROVED** [review]
